src/lib/Form/UiFormFactory.php

```diff
@@ -82,4 +91,15 @@ public function createLocationsActionForm(array $locations = [])
 
         return $this->formFactory->create(LocationActions::class, $data);
     }
+
+    public function createLocationOrderingForm(Location $location)
+    {
+        $data = $this->orderingMapper->mapToForm($location);
+
+        return $this->formFactory->create(
+            Ordering::class,
+            $data,
```

Comment:
As this is already being called on a multi line, just wondering if rather than setting `$data` you could call `mapToForm` directly here. Should still be pretty readable